### PR TITLE
Enable systemd-boot support (jsc#PED-1906)

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by lnussel** [comment]
ping?

**Comment by lnussel** [comment]
@lslezak ?

**Comment by schubi2** [comment]
I have talked to Dominique Leuenberger. He has no issue to enable systemd-boot for openSUSE.
He has also mentioned that we have to take care about openQA in order to fix needles and have to write a testcase which includes systemd-boot. 
Fabian Vogt will take care about openQA and systemd-boot. He has already written a testcase for MicroOS.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+control.dud: control/control.xml
+	mkdud -c control.dud -d tw control
```
Comment (jreidinger) [inline Makefile:2]: well, if we create such file, we should probably also have it in `.gitignore`. But it is not critical.

**Review by jreidinger — APPROVED** [review]

**Comment by schubi2** [comment]
Thank you Josef !

**Pull request merged**